### PR TITLE
Add guac package.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] REQUIRED - The package is added to `packages.txt`

### CODE REVIEW

File: guac.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package:
+  name: guac
+  version: 0.1.0
+  epoch: 0
+  description: Autoscaling components for Kubernetes
```
Comment (ajayk) [inline guac.yaml:5]: Description needs to change to reflect guac

File: guac.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/guacsec/guac
+      tag: v${{package.version}}
+      expected-commit: 01984331b7337492b833d6b96ddb020d56654e39
+
+  - uses: go/build
```
Comment (imjasonh) [inline guac.yaml:24]: Could we put these in the subpackage pipelines? You could even tidy things up with a `range` if you wanted to.

Comment (dlorenc) [reply guac.yaml:24]: I tried to use a loop/range, but it doesn't appear we support template replacement of range variables inside the inputs to a pipeline :(

**Review by joshrwolf — DISMISSED** [review]

**Review by imjasonh — DISMISSED** [review]

File: guac.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    with:
+      packages: ./cmd/guaccollect
+      output: guaccollect
+      ldflags: -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```
Comment (mattmoor) [inline guac.yaml:28]:
```suggestion
      ldflags: -s -w -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```

File: guac.yaml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+subpackages:
+  - name: guacingest
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```
Comment (mattmoor) [inline guac.yaml:35]:
```suggestion
          ldflags: -s -w -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```

File: guac.yaml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+          output: guacingest
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/bin
+          mv ${{targets.destdir}}/usr/bin/guacingest ${{targets.subpkgdir}}/usr/bin
+
+  - name: guacone
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```
Comment (mattmoor) [inline guac.yaml:46]:
```suggestion
          ldflags: -s -w -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```

File: guac.yaml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+          output: guacone
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/bin
+          mv ${{targets.destdir}}/usr/bin/guacone ${{targets.subpkgdir}}/usr/bin
+
+  - name: guacgql
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```
Comment (mattmoor) [inline guac.yaml:57]:
```suggestion
          ldflags: -s -w -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```

File: guac.yaml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+          output: guacgql
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/bin
+          mv ${{targets.destdir}}/usr/bin/guacgql ${{targets.subpkgdir}}/usr/bin
+
+  - name: guaccsub
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```
Comment (mattmoor) [inline guac.yaml:68]:
```suggestion
          ldflags: -s -w -X github.com/guacsec/guac/pkg/version.Version=${{package.version}}
```

File: guac.yaml (hunk 5 above)
Comment (mattmoor) [reply guac.yaml:46]: still need this

File: guac.yaml (hunk 6 above)
Comment (mattmoor) [reply guac.yaml:57]: same

**Review by imjasonh — APPROVED** [review]

**Pull request merged**